### ISSUE TITLE
[Bug] Settings file not being respected?

### ISSUE THREAD

**Issue by KyleTryon** [benphelps/homepage#1446]
### Description

It appears as though the `settings.yaml` file in my scenario may not be respected, though it is loaded. The setting.yaml file exists within the config folder but it's settings are not reflected, while those of widges.yaml and others _are_.

### Steps to reproduce

 I am running a custom helm chart.

Here is the relevant part of the generated template:
```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: homepage
  namespace: default
  labels:
    app.kubernetes.io/name: homepage
data:
  kubernetes.yaml: |
    mode: cluster
  settings.yaml: |
    
    title: "Homepage | ${DOMAIN_NAME}"
    
  bookmarks.yaml: |
    
    - Social:
        - HackerNews:
            - icon: mdi-alpha-y-box
              href: https://news.ycombinator.com/
```

In this example, the bookmarks section as shown in this example _does_ correctly display on the page, but seemingly nothing from the `settings.yaml` is being respected as far as I can tell, it may have to do with the syntax but I believe it is correct based on what I see in the docs and the fact that bookmarks is working.

If I exec into the pod and check the settings config I see this:

```sh
/app # cat /app/config/settings.yaml 

title: "Homepage | x.com"
```

However, in the application we can see the title is unchanged: 
<img width="200" alt="image" src="https://user-images.githubusercontent.com/33272306/235574017-2c4c1ca8-2bbb-4c0f-9d74-ca7bd43d0c50.png">

This is true of any setting change I make to the `settings.yaml` file currently, including layout.

### homepage version

a798c02153df67633b2ac023f5173b1761ac8e3e0cd562eb1e6d62ee4ecf73df

### Installation method

Other (please describe above)

### Container Logs

No errors

### Before submitting, I have made sure to

- [X] Check [the documentation](https://gethomepage.dev/)
- [X] Follow [the troubleshooting guide](https://gethomepage.dev/en/more/troubleshooting/) (please include output above if applicable).
- [X] Search [existing issues](https://github.com/benphelps/homepage/search?q=&type=issues) and [discussions](https://github.com/benphelps/homepage/search?q=&type=discussions).

**Comment by shamoon** [comment]
I think this has to do with the deployment, though Im not certain. There was a known issue that was solved #576 by having the docker-entrypoint.sh set a `HOMEPAGE_BUILDTIME` env. Perhaps you can raise this issue over there https://github.com/jameswynn/helm-charts/tree/main/charts/homepage . We can re-address here if needed

**Comment by KyleTryon** [comment]
@shamoon I have discovered the fix for issue of the `settings.yaml` file not being loaded. I would still consider this a bug though it's possible this was intentional (but slightly hard to discover). I now show the settings.yaml file properly loaded, only after _manually_ clicking the Refresh button in the bottom right of the application. I assumed (like the other config files: bookmarks, services, etc) that settings would be loaded automatically with the Docker container.

<img width="223" alt="image" src="https://user-images.githubusercontent.com/33272306/235934777-b89ba174-b007-4949-8da2-77ac515ac28f.png">

I did not yet dig into where this is read, but I would have imagined this would update automatically like the other configurations.

**Comment by KyleTryon** [comment]
I also notice after every helm deploy I need to refresh again to get the settings to load.

**Comment by shamoon** [comment]
With docker you should not need to hit the reload button after eg recreating the container

**Comment by KyleTryon** [comment]
Ah I see, maybe the issue is I am not creating the container, I am mounting into a pre-built container, `ghcr.io/benphelps/homepage`. Wouldn't this mean the main unofficial chart should have the same issue? Given the other configuration files besides `settings` do not have this limitation, do you think it would be worth reopening this issue or a new one to see if we can get mounted settings files to load on container start?

**Comment by shamoon** [comment]
Sorry I should have said restart or re-create, any time docker-entrypoint is run

That sounds like an issue with the helm chart, which is not part of this codebase

**Comment by budimanjojo** [comment]
But why would it needs to be writeable though? What is the program writing to the config file?
